src/t_zset.c

```diff
@@ -472,18 +489,19 @@ static unsigned long zslDeleteRangeByLex(zskiplist *zsl, zlexrangespec *range, h
 
     x = zsl->header;
     for (i = zsl->level - 1; i >= 0; i--) {
-        while (x->level[i].forward && !zslLexValueGteMin(x->level[i].forward->ele, range)) x = x->level[i].forward;
+        while (x->level[i].forward &&
+               !zslLexValueGteMin(zslGetNodeElement(x->level[i].forward), range)) x = x->level[i].forward;
```

Comment:
A two-lines while loop without braces is slightly hard to read. I suggest expanding to this:

```suggestion
        while (x->level[i].forward &&
               !zslLexValueGteMin(zslGetNodeElement(x->level[i].forward), range)) {
            x = x->level[i].forward;
        }
```

Reply:
I've fixed.